app/presenters/nsm/cost_summary/disbursements.rb

```diff
@@ -15,6 +14,8 @@ def rows
           [
             { text: translated_text(disbursement), classes: 'govuk-table__header' },
             { text: NumberTo.pounds(disbursement.total_cost_pre_vat || 0), classes: 'govuk-table__cell--numeric' },
+            { text: NumberTo.pounds(disbursement.vat || 0), classes: 'govuk-table__cell--numeric' },
```

Comment:
Should we only show the VAT column if VAT is applicable to the claim?

Reply:
Ah, weird quirk here: disbursements can have VAT even if the firm is not VAT registered (unlike work items, letters and calls and fees etc). I think it's because what matters here is whether the provider of the service charges VAT).  So even if the claim is non-vat-applicable in general, VAT can still be a thing here.